src/coreclr/vm/jitinterface.cpp

```diff
@@ -3086,24 +3086,28 @@ void CEEInfo::ComputeRuntimeLookupForSharedGenericToken(DictionaryEntryKind entr
     // Unless we decide otherwise, just do the lookup via a helper function
     pResult->indirections = CORINFO_USEHELPER;
 
-    // Runtime lookups in inlined contexts are not supported by the runtime for now
-    if (pResolvedToken->tokenContext != METHOD_BEING_COMPILED_CONTEXT())
+    MethodDesc* pContextMD = GetMethodFromContext(pResolvedToken->tokenContext);
+    if (pContextMD == nullptr)
     {
+        // Class context is not yet supported
```

Comment:
What's missing for this?

Reply:
Working on it, should be this VM function specific change.
+ need to unmark a couple of trees as invariant in jit